animation_nodes/nodes/rotation/c_utils.pyx

```diff
@@ -121,3 +124,153 @@ def randomQuaternionList(int seed, int amount):
         quaternionNormalize_InPlace(result.data + i)
 
     return result
+
```

Comment:
Create a cython function that does the conversion for a single element in the math module. Then use that function here. This goes for all conversions.